test/location_search_test.dart

```diff
@@ -13,7 +13,7 @@ void main() {
 
       await tester.tap(find.byIcon(Icons.search));
       await tester.pumpAndSettle();
-      expect(find.text("SGW Campus, Montreal"), findsOneWidget);
+      expect(find.text("HALL BUILDING"), findsOneWidget);
```

Comment:
still failing. we should rewrite this test.

Reply:
I disabled the test for now, we'll have to remake the test to fit our current implementation.

Reply:
That's fine